tofcamera_mesasr.orogen

```diff
@@ -12,6 +12,12 @@ using_library "vizkit3d"
 task_context "Task" do
    needs_configuration
 
+  property('interface', 'std::string', 'ETHERNET').
+	 doc ('Type of interface of the camera sensor. So far Swiss Ranger Cameras are available with USB or Ethernet interfaces for data connection. Possible values: USB, ETHERNET')
+
```

Comment:
I recomment to use a URI based format like:
serial://
ip://

and parse this within one 'port' member for deice access.
Otherwise it could confuse on the interface.

Otherwise use a enum for the type, and make sure ip is only set when ETHERNET is set, and vice-versa for the serial port.